### DISCUSSION TITLE
Error Indication Spelling Should Match the Command

### DISCUSSION THREAD

**Discussion by Prab-27** [pypa/pipx#1489] [Ideas]
Hi all, when I run the `cowsay` app and encounter incorrect syntax, it displays the following usage message: `usage: Cowsay [-h] [-c CHARACTER] -t TEXT [-v].` I believe there should be a lowercase ‘c’ in ‘cowsay,’ because, as I understand it, the intention is to use ‘cowsay.’ If you agree, I’d like to open my first issue here.

**Comment by dukecat0** [comment]
This issue is not related to pipx. Please open an issue in cowsay project instead.

**Reply by Prab-27** [reply]
ya Thank you , You are right  . I am a beginner so made a mistake .I am Sorry
